_posts/2020-06-21-covid-decision.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: Will The Conference Go Ahead In 2020?
```

Comment:
I think the title should make it clear the event is postponed and the description should be why you have come to that decision. Having to click through to find the answer could be a bit frustrating.